src/Reflection/SignatureMap/functionMap.php

```diff
@@ -7178,7 +7178,7 @@
 'mysqli::next_result' => ['bool'],
 'mysqli::options' => ['bool', 'option'=>'int', 'value'=>'mixed'],
 'mysqli::ping' => ['bool'],
-'mysqli::poll' => ['int|false', '&read'=>'array', '&error'=>'array', '&reject'=>'array', 'sec'=>'int', 'usec='=>'int'],
+'mysqli::poll' => ['int|false', '&w_read'=>'array', '&w_error'=>'array', '&w_reject'=>'array', 'sec'=>'int', 'usec='=>'int'],
```

Comment:
Does it create new variable (like `$matches` in preg_match) or does it just read the argument? The manual isn't clear on this...

Reply:
Here a test I made (executed with PHP 7.3.4):
```
$ php -r 'mysqli::poll($a, $b, $c, 1); var_dump($a, $b, $c);'
PHP Warning:  mysqli::poll() expects parameter 3 to be array, null given in Command line code on line 1
NULL
NULL
NULL
```
Even if the behavior is not satisfactory there is no notice for undefined variables, so I conclude that the function creates them.

Reply:
>  * 1. '&rw_name' indicates that a parameter with a value is expected to be passed in, and may be modified.
>  * 2. '&w_name' indicates that a parameter is expected to be passed in, and the value will be ignored, and may be overwritten.

By re-reading this I think the right prefix is `&rw_` because the values of these 3 variable are not meant to be ignored.
What's your opinion on this?

Reply:
If the variable does not have to exist, then it's `&w`. If it has to exist and it's read, then it has to be `&rw`. `&rw` has no special handling in PHPStan because it's not necessary.

Reply:
Ok, so I let the code as it is.

Reply:
@ondrejmirtes is it ok for you or do you expect something else?

Reply:
It's okay I think :)